graphite/graphite.go

```diff
@@ -100,6 +101,15 @@ func readTraceback(resp *http.Response) (*[]string, error) {
 // DefaultClient is the default HTTP client for requests.
 var DefaultClient = &http.Client{
 	Timeout: time.Minute,
+	Transport: &http.Transport{
+		MaxIdleConnsPerHost: 0,
```

Comment:
i can confirm that even with `MaxIdleConnsPerHost: -1` i'm still seeing 
 graphite RequestError (...): read tcp ...:80: use of closed network connection